modules/microshift-enable-sigstore-mirror-registries.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * microshift/microshift_auth_security/microshift-verify-container-signatures.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-enable-sigstore-mirror-registries_{context}"]
+= Enabling sigstore attachments for mirror registries
+
+If you are using mirror registries you must apply additional configuration to enable sigstore attachments and mirroring by digest.
+
+.Prerequisites
+
+* You have admin access to the {microshift-short} host.
+* You completed the steps in "Verifying container signatures using sigstore."
+
+.Procedure
+
+. Enable sigstore attachments by creating the `/etc/containers/registries.d/mirror.registry.local.yaml` file.
++
+[source,terminal]
+----
+$ cat /etc/containers/registries.d/mirror.registry.local.yaml <1>
+docker:
+   mirror.registry.local:
+        use-sigstore-attachments: true
+----
+<1> Optional: Replace `mirror.registry.local.yaml` with your mirror registry URL and filename.
```

Comment:
This is not optional. It's most likely that user's registry has another URL.
Let's separate file and URL renames. Please, add <2> on line 24.

```suggestion
<1> Name `mirror.registry.local.yaml` file after your mirror registry URL.
<2> Replace `mirror.registry.local` with your mirror registry URL.
```